### Waypoints ###

waypoint_manager: {

waypoints:
 [[0, 1, -1.5, 0],
  [0, -1, -1.5, 0]],
  #[-1, 1, -1.5, 0]],
  #[[0, 0, -1.5, 0]],

threshold: 0.1,
cyle: True
}

### Controller Parameters

controller: {

x_dot_P: 0.5,
x_dot_I: 0.0,
x_dot_D: 0.05,

y_dot_P: 0.5,
y_dot_I: 0.0,
y_dot_D: 0.05,

z_dot_P: 1.0,
z_dot_I: 0.0,
z_dot_D: 0.0,

x_P: 1.5,
x_I: 0.0,
x_D: 0.4,

y_P: 1.5,
y_I: 0.0,
y_D: 0.4,

z_P: 0.6,
z_I: 0.25,
z_D: 0.1,

psi_P: 1.0,
psi_I: 0.0,
psi_D: 0.0,

tau: 0.05,

equilibrium_thrust: 0.65,

max_roll: 0.196,
max_pitch: 0.196,
max_yaw_rate: 0.785,
max_throttle: 0.85,
max_x_dot: 0.5,
max_y_dot: 0.5,
max_z_dot: 0.5

}
